### PR TITLE
Merge pull request #1 from jcy2704/youtube-1

### PR DESCRIPTION
**Pull request by alejandrotoledoweb**
Youtube Basic HTML & CSS Structure

### CODE REVIEW

**Review by teekaytech — CHANGES_REQUESTED** [review]
# Status :Invalid Review Request :heavy_multiplication_x: 

**Hi @jcy2704 and @alejandrotoledoweb, congratulations on submitting your first code review request. Unfortunately, I cannot review your project.**

* * *
- [ ] **You should use Github Flow in this project.**


- You created your pull request from the feature-branch (in your case, youtube-1) to the master branch.
This is wrong. Your master branch should only have a single commit (initial commit) while all other commits are to be done on the feature branch (youtube-1 in your case).
- For better understanding, please take a step back and read a lesson [here](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/85215964/). Also, watch [the video](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/85215964/) at the end of this lesson. 
- You should make sure that your repo looks professional before review. Please take a step back and read a lesson [here](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/). Then apply 3 simple rules from this lesson (it will be easy as you can use a template provided there).
- **You will have to create a brand new repo with the correct setup and feature branch (with DESCRIPTIVE name) and copy your code there.**

### Please note that this Code Review Request will be marked as invalid, so please submit a new one once you are ready!

* * *

In summary, do not feel bad. Things can be a bit tricky at first, but believe me, it gets better with time.


*__You can reach out to me on slack (@taofeekolalere) or GitHub (@teekaytech).__*

**Happy coding. :man_technologist:**

**Comment by jcy2704** [comment]
Hello, is it because we merged the pull request to the master branch? I press the merge button by mistake thinking it was just something I needed to click, however, is my bad I didn't read. Do we need to start from scratch and do commit by commit or just paste the code to a new repository and do one commit and make the pull request?
